pkg/pipelines/parameters_evaluator.go

```diff
@@ -4,16 +4,17 @@ import (
 	"fmt"
 	"strconv"
 
+	"github.com/Jeffail/gabs/v2"
 	consolelogger "github.com/semaphoreci/spc/pkg/consolelogger"
-	parameters "github.com/semaphoreci/spc/pkg/parameters"
+	templates "github.com/semaphoreci/spc/pkg/templates"
```

Comment:
With this change, we are no longer using the parameters package, so let's remove it. 
Also, we can then rename `parameteres_evaluator` into `templates_evaluator` to maintain clarity and we should also do the same change [here](https://github.com/semaphoreci/spc/blob/master/pkg/cli/compile.go#L23-L31).